contracts/zero-ex/contracts/src/features/interfaces/IMultiplexFeature.sol

```diff
@@ -46,6 +46,8 @@ interface IMultiplexFeature {
         bool useSelfBalance;
         // The recipient of the bought output tokens.
         address recipient;
+        // The sender of the transaction.
+        address msgSender;
```

Comment:
Is this the beneficiary of the metatransaction, or the relayer? We should invent new terminology here so that it's clear.

Reply:
In a discussion we had earlier today, @duncancmt and @abls and I felt that the best naming would be payer (// the sender of the input token), which kinda mirrors what's going on with recipient - the receiver of the output token.

Reply:
@dekz I see what you mean. I think this gets a little confusing with OTC/RFQ when the flow of value is bidirectional (equivalently: when the operation being executed is effectively a metatransaction with 2 signers). I concede that this isn't a big deal for the present PR, though.